src/Symfony/Component/Workflow/SupportStrategy/WorkflowSupportStrategyInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\SupportStrategy;
+
+use Symfony\Component\Workflow\WorkflowInterface;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+interface WorkflowSupportStrategyInterface
+{
+    /**
+     * @param WorkflowInterface $workflow
+     * @param object            $subject
+     *
+     * @return bool
```

Comment:
Sorry about asking again some change, but we don't need this PHP doc. Could you remove it?
Thanks

Reply:
Don't be sorry, that's what the PR are made for ;)